code/CLI/CommandData.py

```diff
@@ -193,6 +193,16 @@
             "skip_line" : False
         }
     ],
+    "rename_parameter" : [
+        {
+            "usage" : "rename_parameter <method_name> <old_param_name> <new_param_name>",
```

Comment:
Rename parameter should include the class name as well

Reply:
I will make the changes, Thank you!